matRad_calcPhotonDose.m

```diff
@@ -116,15 +116,15 @@
 % set up convolution grid
 if strcmp(num2str(pln.bixelWidth),'field')
     % get data from DICOM import
-    intConvResolution = pln.Collimation.ConvResolution; 
-    fieldWidth = pln.Collimation.FieldWidth;
+    intConvResolution = pln.Collimation.convResolution; 
+    fieldWidth = pln.Collimation.fieldWidth;
 else
     intConvResolution = .5; % [mm]
     fieldWidth = pln.bixelWidth;
 end
 
 % Make a 2D grid extending +/- lateral cutoff for kernels
-intConvLimits = lateralCutoff + ceil(fieldWidth/(2*intConvResolution))*intConvResolution; % [mm]
+intConvLimits = lateralCutoff; % [mm]
```

Comment:
@AlexS1992 I figure this is should not be changed...

--> I reversed this change in my latest commit https://github.com/markbangert/matRad/commit/7110e0e1c499d98a072e0853cd98854c9a361f97